fs/span-manager/span_manager.go

```diff
@@ -162,6 +163,49 @@ func (m *SpanManager) buildAllSpans() {
 	}
 }
 
+// FetchSpanBackground invokes the reader to fetch the span in the background and cache it.
+// It is invoked by the backgroundFetcher.
+func (m *SpanManager) FetchSpanInBackground(spanID soci.SpanID) error {
```

Comment:
Doesn't it block until the span is fetched? We don't have to mention "Background" if so.

Reply:
Renamed.